app/main.py

```diff
@@ -2,0 +1,21 @@
+def copy_file(command: str) -> None:
+    commands = command.split()
+    if (
+        len(commands) < 3
+        or commands[0] != "cp"
+        or commands[1] == commands[2]
+    ):
+        return
+    try:
+        with (
+            open(commands[1], "r") as source,
+            open(commands[2], "a") as target
+        ):
+            for line in source:
+                target.write(line)
+    except UnicodeError:
+        print("Error while copy file")
```

Comment:
Clean Code: It's a good practice to handle exceptions in your code. However, the error message 'Error while copy file' is not very informative. Consider providing more context in the error message, such as the file names and the type of error.